akka-remote/src/main/scala/akka/remote/serialization/DaemonMsgCreateSerializer.scala

```diff
@@ -131,6 +123,25 @@ private[akka] class DaemonMsgCreateSerializer(val system: ExtendedActorSystem) e
 
   protected def serialize(any: Any): ByteString = ByteString.copyFrom(serialization.serialize(any.asInstanceOf[AnyRef]).get)
 
+  private def serializeToSerializedMessage(any: Any): SerializedMessage = {
+    val m = any.asInstanceOf[AnyRef]
+
+    val serializer = serialization.findSerializerFor(m)
+    val builder = SerializedMessage.newBuilder()
+
+    serializer match {
+      case ser: SerializerWithStringManifest ⇒
+        val manifest = ser.manifest(m)
+        if (manifest != "")
+          builder.setMessageManifest(ByteString.copyFromUtf8(manifest))
+      case _ ⇒
+    }
+
+    builder.setSerializerId(serializer.identifier)
+      .setMessage(ByteString.copyFrom(serializer.toBinary(m)))
+    builder.build()
+  }
+
```

Comment:
Pretty much exactly this code is in at least 3 places of the code base now, but with different proto-types being built, could maybe warrant a utility in serialization for doing it (and take the cost of a tuple3 alloc)